### DISCUSSION TITLE
SML - no MQTT message to iobroker

### DISCUSSION THREAD

**Discussion by Himan71** [arendst/Tasmota#13528] [Q&A] [answered]
Hello,
i'm using Tasmota 9.5.0.9 with an ESP8266 and want to read out my home EHZ and transfer the power values via MQTT to a raspberry pi iobroker.
I compiled it using the following configuration:
#define USE_SCRIPT
#define  USE_SCRPIT_JSON_EXPORT
#define USE_SCRIPT_SUB_COMMAND
#define USE_SML_M

decoding of EHZ data works well, in the webUI the values are visible

in the console output i can see MQT messages: e.g.
16:14:54.379 MQT: tele/tasmota_11D7EF/SENSOR = {"Time":"2021-10-30T16:14:54","Haus  ":{"Power_all":607833.0,"Power_curr":508,"Power_avg":607.833}}  see screenshot below

I configured the MQTT Host with ip of raspberry, port is 1883, user name and pw are set as well

I log the network traffic with wireshark but i never see any telegram from the SENOR to the pi
Normally i expect, that these telegrams are visible to that maybe the topic or something else is wrong, no telegram at all, only console messages...

i checked many documentations, examples, etc. but i didn't find anything which fixes this issue.

Is it a wrong configuration from my side, or a problem of the self compiled binary, or maybe a bug of this release?

I would be verry happy if somebody can help me on this topic,.

Greetings, 
Matthias

![Konsole](https://user-images.githubusercontent.com/41704859/139539395-f398f5db-6f78-42c3-8331-575446f05ba5.png)
![overview](https://user-images.githubusercontent.com/41704859/139539396-c9cdc402-6a15-4e5e-af34-c5ae8073395e.png)

**Comment by barbudor** [comment]
You device is connected to MQTT so publish should be working unless your have ACL rules on your broker

Note If you're running wireshark on another pc and not on the pi you will not see any unicast messages between the device and the pi.
You should better use a mqtt subscription or any other mqtt tool like mqtt explorer to see that.

**Comment by Himan71** [comment, marked as answer]
Many thanks for your fast replay. Thinking about a few seconds on your comment regarding wirekshark, mia culpa, you are absolutely right.  I try out MQTT Explorer and will see, many thanks!!

**Reply by barbudor** [reply]
On the pi you can use command line tcpdump
'sudo tcpdump -i any port 1883'

**Reply by Himan71** [reply]
ok, packets are landing, now i check the topic... Thanks again!!
